testing/geoip-database/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=geoip-database
+pkgver=20170714
+pkgrel=0
+pkgdesc="GeoIP library (legacy country databases)"
+url="http://maxmind.com"
+arch="noarch"
+license="CC-4.0"
+depends=""
+options="!check"
+source="http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz
```

Comment:
not much can be done about that - & is why the Debian version number is also a date